onnxruntime/core/providers/rocm/reduction/reduction_ops.cc

```diff
@@ -387,7 +278,7 @@ Status PrepareForReduce(const Tensor* X,
   }
 
   const auto input_dims = input_shape.GetDims();
-  InlinedVector<bool> reduced(rank, false);
+  std::vector<bool> reduced(rank, false);
```

Comment:
Any reason to change InlinedVector to std::vector?

Reply:
Symmetry with CUDA EP, but I can change this back if desired.

Reply:
InlinedVector is better in this case.
We can leave it now for this PR.
Please change both rocm and cuda to InlinedVector in future PR in you have time.